shortfin/CMakeLists.txt

```diff
@@ -237,6 +245,61 @@ else()
 endif()
 shortfin_pop_bundled_lib_options()
 
+################################################################################
+# Tokenizer Library
+################################################################################
+
+function(shortfin_check_tokenizers)
+  # Make sure that rust/cargo is installed and usable.
+  find_program(SHORTFIN_CARGO_PATH NAMES cargo NO_CACHE)
+  if(NOT SHORTFIN_CARGO_PATH)
+    message(SEND_ERROR
+      "Building with -DSHORTFIN_ENABLE_TOKENIZERS=ON requires cargo (Rust's build tool). "
+      "Please follow Rust documentation to install. On Ubuntu, this can typically be accomplished with:\n"
+      "  sudo apt install rustup && rustup default stable"
+    )
+  endif()
```

Comment:
These docs? https://www.rust-lang.org/tools/install

Can you include that link in the message?

Reply:
No idea - I didn't read any docs. Just followed the breadcrumbs of things that the tools said when a step hadn't been done.